contrib/for-tests/volumes/nfs/run_nfs

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
As per previous comment, andy reason not to add a ".sh" extension to this file name?